Server/src/routes/authenticationRouter.ts

```diff
@@ -22,6 +23,11 @@ router.post('/login', (request: Request, response: Response, next: NextFunction)
 //TODO: Implement when doing password reset 
 router.get('/resetPassword', (request: Request, response: Response, next: NextFunction) => {
 
-})
+});
+
+router.get('/userDetails', JWTAuthenticator.verifyJWT, async (request: Request, response: Response, next: NextFunction) => {
```

Comment:
would you be able to give me a rundown on how this authenticator functions?

Reply:
Yes anytime. We can do it on discord